docs/julia.rst

```diff
@@ -439,3 +441,69 @@ Backend: eWASM
 --------------
 
 TBD
+
+Specification of JULIA Object
+=============================
+
+Grammar::
+
+    TopLevelObject = 'object' '{' Code? ( Object | Data )* '}'
+    Object = 'object' StringLiteral '{' Code? ( Object | Data )* '}'
+    Code = 'code' JuliaBlock
+    Data = 'data' StringLiteral HexLiteral
+    HexLiteral = 'hex' ('"' ([0-9a-fA-F]{2})* '"' | '\'' ([0-9a-fA-F]{2})* '\'')
+    StringLiteral = '"' ([^"\r\n\\] | '\\' .)* '"'
+
+Above, ``JuliaBlock`` refers to ``Block`` in the JULIA code grammar explained in the previous chapter.
+
+An example JULIA Object is shown below:
+
+..code::
+
+    // Code consists of a single object. A single "code" node is the code of the object.
+    // Every (other) named object or data section is serialized and
+    // made accessible to datacopy / dataoffset / datasize
+    object {
+        code {
+            let size = datasize("runtime")
```

Comment:
We should add a comment here telling that `datasize` is a special built-in function that references the data below.